src/style.css

```diff
@@ -0,0 +1,102 @@
+@font-face {
+  font-family: Avenir;
+  font-weight: 300px;
+  src: url(fonts/Avenir-Book.ttf);
+}
+
+@font-face {
+  font-family: Avenir;
+  font-weight: bold;
+  src: url(fonts/Avenir-Heavy.ttf);
+}
+
+:root {
+  --main-text-color: #3d4e61;
+}
+
+.html {
+  box-shadow: 0 1px 8px #ecedef
+}
+
+.body {
+  margin-top: 0;
+}
+
+.container {
+  margin-top: 20px;
+  position: relative;
+}
+
+.search-field {
+  color: var(--main-text-color);
+  width: 100%;
+  height: 70px;
+  border-radius: 4px;
+  border: solid 1px #e1e7ed;
+   font-size: 16px;
+  font-family: Avenir, sans-serif;
```

Comment:
move `font-family` to `html`
we usually store common font styles in `html`
```
html {
  font-family: Avenir, sans-serif;
}

Reply:
when i move font-family to html, it`s not wurking.